mixer/adapter/inventory.yaml

```diff
@@ -0,0 +1,9 @@
+denier: "istio.io/istio/mixer/adapter/denier"
```

Comment:
Is this a generated file?

Reply:
This should not be. IDTS

Reply:
IDTS. This is the only place where adapters are now declared.

Reply:
@geeknoid it is not generated. it replaces the skylark script for declaring which adapters you wish to build into mixer.